draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -175,9 +175,12 @@ values:
 
 "integrity"
 
-: A string (Section 3.7 of {{!I-D.ietf-httpbis-header-structure}}) containing
-  the lowercase name of the response header field that guards the response
-  payload's integrity.
+: A string (Section 3.7 of {{!I-D.ietf-httpbis-header-structure}}) containing a
+  "/"-separated sequence of names starting with the lowercase name of the
```

Comment:
May not be worth deciding now, but future integrity formats may want names containing a variable number of slashes (e.g. a pair of base64-encoded strings), so you may want an escaping mechanism eventually.

Reply:
IIUC, the string here refers to format id such as "digest/mi-sha256-03" so I think we are OK?

Reply:
Yeah, I think this is fine for the foreseeable future; I was just hypothesizing about a new integrity header.